src/libmongoc/doc/mongoc_database_find_collections.rst

```diff
@@ -43,5 +43,9 @@ Errors are propagated via the ``error`` parameter.
 Returns
 -------
 
-A cursor where each result corresponds to the server's representation of a collection in this database.
+.. include:: includes/returns-cursor.txt
+
+In the returned cursor each result corresponds to the server's representation of a collection in this database.
+
+The cursor functions :symbol:`mongoc_cursor_set_limit`, :symbol:`mongoc_cursor_set_batch_size`, and :symbol:`mongoc_cursor_set_max_await_time_ms` have no use on the returned cursor.
```

Comment:
I see a similar paragraph in src/libmongoc/doc/mongoc_database_find_collections_with_opts.rst and I have included it under the understanding that both functions behave the same way.

However, if I'm wrong, just tell me and I'll remove the paragraph.

Reply:
Adding this paragraph is a good idea. This internally calls `mongoc_database_find_collections_with_opts`. The only difference is that this returns `NULL` if an error occurred.